python/pfs/pipe2d/generateCommands.py

```diff
@@ -1428,18 +1429,23 @@ def generateCommands(
 
     logger = logger.getChild("generateCommands")
 
-    initSource, calibBlocks, scienceBlocks = processYaml(specFile)
-    possibleBlocks = unique(list(calibBlocks.keys()) + list(scienceBlocks.keys()))
-    if blocks is None:
-        blocks = possibleBlocks
-
     dataDir = os.path.abspath(dataDir)
     if not os.path.exists(dataDir):
         if force:
             logger.warning("'%s' doesn't exist", dataDir)
         else:
             raise RuntimeError(f"'{dataDir}' doesn't exist")
 
+    if calib is None:
+        calib = os.path.join(dataDir, "CALIB")
+    else:
+        calib = os.path.abspath(calib)
```

Comment:
Maybe check if the `calib` directory exists, and create it if it doesn't (or at least warn)?

Reply:
I added a check "`--init` is on or `exists(calib)`". (If `--init` is on, `calib` need not exist because the generated commands start with ingesting initial detectorMaps)